driver-async/src/main/com/mongodb/async/client/AbstractSubscription.java

```diff
@@ -36,7 +37,8 @@
     private boolean isTerminated = false;
     /* protected by `this` */
 
-    private final ConcurrentLinkedQueue<TResult> resultsQueue = new ConcurrentLinkedQueue<TResult>();
+    private static final Object NULL_PLACEHOLDER = new Object();
```

Comment:
Move to the top next to the other static.

Reply:
Done